platform/core/Makefile

```diff
@@ -0,0 +1,53 @@
+export_dir_ios = export/ios
+ios_archs := aarch64-apple-ios aarch64-apple-ios-sim x86_64-apple-ios
+ios_targets := iPhone simulator
+build_type := $(if $(filter $(MODE), release),release,debug)
+
+build_ios_xcframework:
+# Ensure correct OS is used
+ifeq ($(OS),Windows_NT)
+	@echo "xcframework can be built only in macOS"
+	exit 1
+else
+ifneq ($(shell uname -s),Darwin)
+	@echo "xcframework can be built only in macOS"
+	exit 1
+endif
+endif
+
+	# Workspace setup
+	rm -rf $(export_dir_ios)
+	mkdir -p $(export_dir_ios)/iPhone
+	mkdir -p $(export_dir_ios)/simulator
+
+	# Check rustup is installed
+	if ! hash rustup &> /dev/null; then echo "To add Rust in your local envirnonment execute: curl https://sh.rustup.rs -sSf | sh"; exit 1; fi
```

Comment:
Installing rust should probably be a Makefile target too.

Reply:
Could be, but I think that by default Rust installation has some configuration steps where the developer has to explicitly opt in/out.

Reply:
Is there not a way to include reasonable defaults? At any rate, being a target does not mean it's going to be unattended, it's just simpler to tell them to run make install-rust than make them copy and paste that into their command line.

Reply:
I did not investigate if possible to add reasonable defaults, I'll check it. But the general idea is to tell developers to install Rust on their system, not necessarily to exactly run that shell script.

Reply:
OK, then minor :memo: nit: say something like "Install Rust in your local environment, for instance via... "